packages/apps/dashboard/server/src/modules/networks/networks.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import {
+  Controller,
+  Get,
+  HttpCode,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import { NetworksService } from './networks.service';
+import { ChainId } from '@human-protocol/sdk';
+
+@ApiTags('Networks')
+@Controller('/networks')
+@UsePipes(new ValidationPipe({ transform: true }))
+export class NetworksController {
+  constructor(private readonly networksService: NetworksService) {}
+
+  @Get('/available-networks')
```

Comment:
```suggestion
  @Get('/operating')
```
You already have `networks` in `@Controller`, so full path is `networks/available-networks` and looks a bit excessive